django_react_proj/users-fe/package.json

```diff
@@ -18,6 +18,8 @@
     "moment": "^2.29.1",
```

Comment:
iirc you don't use`moment` anymore. Remove this dependency.

Reply:
I will do it